pandas/tests/plotting/test_datetimelike.py

```diff
@@ -217,6 +217,7 @@ def test_line_plot_period_mlt_frame(self, frqncy):
         freq = df.index.asfreq(df.index.freq.rule_code).freq
         _check_plot_works(df.plot, freq)
 
+    @pytest.mark.filterwarnings(r"ignore:PeriodDtype\[B\] is deprecated:FutureWarning")
```

Comment:
do we need to find a workaround in plotting before doing this?

Reply:
yes.  suppressing the warning on this test is OK, but i think we need a better solution than suppressing here https://github.com/pandas-dev/pandas/pull/53511/files#diff-9e7653bee8c4fb4e42459af383020b7d754df6c55e79680d20e81d588d51f029R283